test/fixtures/local_config/.np-config.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I think all the fixtures for the config tests should be nested inside `test/fixtures/config`. Then this could be renamed to just `local`.

Reply:
@sindresorhus 
Done. Now all fixtures are bundled in `test/fixtures/config`